ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -541,6 +541,16 @@ object BigDiffy extends Command with Serializable {
     }
   }
 
+  // a null TableFieldSchema mode can be treated as "NULLABLE", which is the
+  // default value according to the docs
+  private def getNormalizedFieldMode(f: TableFieldSchema): String = {
```

Comment:
nitpick, is this really "normalized"?

Reply:
NP, I don't mind renaming the function, however normalized in this case makes sense to me with the comment. Did you have a more descriptive name in mind?

Reply:
getFieldModeWithDefault maybe